packages/fetchai/skills/aries_alice/behaviours.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This package contains the behaviour for the aries_alice skill."""
+
+from aea.skills.behaviours import OneShotBehaviour
```

Comment:
It seems an overkill to have a behaviour for a single logger statement. You can print the same from the handler.

Reply:
As a cleanup, I planned to have a single `admin_post` in the behaviour and have the handlers call it, as oppose to having `admin_post` in every handler. Also this removes all the config parameters for the handlers, which are used only by the `admin_post` method.

Reply:
Ok but focus on the code you have not the code you will have. We want to release this today and so its better to drop it imo.

Reply:
Done. Added that in a To-Do GH issue.